<launch>

  <node name="recognizer" pkg="pocketsphinx" type="recognizer.py" output="screen">
    <param name="lm" value="$(find fetch_pbd_speech_recognition)/data/commands.lm"/>
    <param name="dict" value="$(find fetch_pbd_speech_recognition)/data/commands.dic"/>
  </node>

  <node name="command_recognizer" pkg="fetch_pbd_speech_recognition" type="recognize_commands.py" output="screen"></node>

</launch>
